client/checkout/api/index.js

```diff
@@ -444,6 +444,45 @@ export default class WCPayAPI {
 			} );
 	}
 
+	/**
+	 * Confirm Stripe payment with fallback for rate limit error.
+	 *
+	 * @param {Object} confirmParams Confirm payment request parameters.
+	 * @param {Object|StripeElements} elements Stripe elements.
+	 * @param {string|null} paymentIntentSecret Payment intent secret used to validate payment on rate limit error
+	 *
+	 * @return {Promise} The final promise for the request to the server.
```

Comment:
It's not used by the server afterward, it's just `The payment confirmation promise` (that is resolved only in case of an error, which would then be thrown right after and shown on the checkout page, with no interaction with the server)

Reply:
I forgot to update it. I will fix it.

Reply:
Fixed.